### PR TITLE
Prison changes

### PR DESCRIPTION
**Pull request by Spyroshark**
## About The Pull Request

This adds a moving monorail to prison, and adds a different ocean tile that can kill you. A couple areas have been slightly changed, too.

## Why It's Good For The Game

The change from prison being in space to being above the ocean was kind of lazy, and this adds some new mechanics to keep prison interesting.

## Changelog
:cl:
add: You can now fall into the ocean on prison, and there is a working monorail!
/:cl:

### CODE REVIEW

**Comment by spookydonut** [comment]
kill turfs are lazy and horrible for players

**Comment by Spyroshark** [comment]
People don't like this PR, i'm just gonna close it instead of arguing.

**Comment by spookydonut** [comment]
@Spyroshark merge conflict

**Comment by Spyroshark** [comment]
Fixed the merge conflict, and i redid the map changes, and I left the indestructable walls unchanged, since people didn't really like that part.

File: code/game/turfs/open.dm [hunk 1]
```diff
@@ -102,6 +102,50 @@
...
+var/list/canfall = typecacheof(list(
+					/mob/living,
+					/obj/vehicle,
+					/obj/machinery,
+					/obj/item,
+					/obj/structure))
+
+var/list/nofall = typecacheof(list(
+					/obj/structure/lattice,
+					/obj/structure/monorail))//just for specific things that should be able to float
```
Comment (nemvar) [inline code/game/turfs/open.dm]: Either indent these or turn them into proper global lists.

Comment (nemvar) [reply code/game/turfs/open.dm]: GLOBAL_LIST_INIT is the proper macro I think. No idea if ocean turfs are common, if they are turn these into globals.

Comment (Spyroshark) [reply code/game/turfs/open.dm]: They are common, I'll do that

**Comment by Rohesie** [comment]
I like this meme.

**Comment by Spyroshark** [comment]
Closing this, I don't really want to finish it. If someone else wants to they can go ahead.

**Pull request closed without merging**